### DISCUSSION TITLE
Problem with "Track Select" command on encoders

### DISCUSSION THREAD

**Discussion by ll-dasein-ll** [KBLiveSolutions/open.control#14] [Troubleshooting] [answered]
I mentioned this issue in an email to Pierre last night but he asked me to log issues in the forum so other users can benefit from discussions and stuff, so here's my first troubleshooting post :)

I noticed that the Track Select function on the encoders doesn't seem to do anything in Arrangement or Session view.  I wasn't sure if I was misunderstanding what it was supposed to do, but it does seem to be an issue in the software, based on the reply I got.

As a workaround I've assigned buttons to "prev track" and "next track" to move up and down my projects, but I can't wait to be able to scroll down/across tracks with an encoder - especially for live performance.

**Comment by KBLiveSolutions** [comment, marked as answer]
The updated Remote Script should fix this issue

https://github.com/KBLiveSolutions/open.control-remote_script/archive/refs/heads/custom-actions.zip

Note that if an encoder was already assigned to this action, you need to re-assign it, since the corresponding CC has changed.

**Comment by ll-dasein-ll** [comment]
Hi Pierre -

Thanks for the script update, but I'm afraid I still can't get it to work.  I've made sure I've re-assigned the command (and I've checked that other commands are working to make sure the script is installed correctly), switched back from other commands to Track Select, but it still doesn't scroll through the tracks in either Arrangement or Session view.  I've tried with both encoders with Hold & Turn, and also just Turn, but it doesn't respond to turning the knob.  Sorry for the bad news!

**Comment by KBLiveSolutions** [comment]
Are you using the latest version of the editor?
https://kblivesolutions.github.io/open.control-editor/
It should show a "crossfader" action just above the "Track Select" action.

**Reply by ll-dasein-ll** [reply]
Ah, that might be it - I'm running the editor locally so it won't be the new one.  Cool, I'll try that and update shortly!

**Reply by ll-dasein-ll** [reply]
That was it!  Sorted - thank you so much! - How can I mark this thread as Answered?

**Reply by GTraux** [reply]
Good question, @ll-dasein-ll Want to this too.

**Reply by KBLiveSolutions** [reply]
I think that's my very own privilege ! 💪
